README.md

```diff
@@ -83,6 +83,11 @@ to run the tests run the following command
 - GitHub: [@ANTOINE1128](https://github.com/ANTOINE1128)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/antoine-makdessy/)
 
+👤: **Tanmayi Manku**
+
+- GitHub: [@tan12082001](https://github.com/tan12082001)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tanmayi-manku/)
+
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![image](https://github.com/ANTOINE1128/Blog-App1/assets/79058364/965a9cc2-3007-4d7d-8cf0-adfe117d3ddc)

- [ ] Please update your Readme, it will be nice to add a link to the license file, as this will make your readme more informative and professional.  ✍🏻